x/blob/types/payforblob_test.go

```diff
@@ -77,14 +77,14 @@ func TestCreateCommitment(t *testing.T) {
 			name:         "blob of 11 shares succeeds",
 			namespace:    bytes.Repeat([]byte{0xFF}, 8),
 			blob:         bytes.Repeat([]byte{0xFF}, 11*ShareSize),
-			expected:     []byte{0x1e, 0xdc, 0xc4, 0x69, 0x8f, 0x47, 0xf6, 0x8d, 0xfc, 0x11, 0xec, 0xac, 0xaa, 0x37, 0x4a, 0x3d, 0xbd, 0xfc, 0x1a, 0x9b, 0x6e, 0x87, 0x6f, 0xba, 0xd3, 0x6c, 0x6, 0x6c, 0x9f, 0x5b, 0x65, 0x38},
+			expected:     []byte{0x9f, 0x44, 0xd5, 0x12, 0xe9, 0x6b, 0xea, 0xb3, 0xf2, 0xfe, 0x7b, 0x46, 0xc6, 0x4c, 0xee, 0x70, 0xb0, 0x86, 0xca, 0x94, 0x7e, 0x1b, 0x95, 0xd2, 0x0, 0x78, 0x32, 0xb5, 0x94, 0x68, 0x67, 0xf0},
```

Comment:
[for reviewers] until this unit test verifies the commitment bytes, these bytes are expected to change whenever we modify how the commitment is generated or the schema of the blob shares